Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class ExperimentIntegrationTests: BaseTestCase {
```

Comment:
Hello! Looking into this code it seems there will be some Swiftlint rules failing. You can check that by running it locally:
1. [Installing](https://formulae.brew.sh/formula/swiftlint) Swiftlint with Brew
2. At root of the project run `swiftlint`, and it will tell you if there's anything to fix (in which file, what line, etc).

Reply:
Updated based on this